plugin/metrics/vars/vars.go

```diff
@@ -65,6 +65,12 @@ var (
 		Name:      "panic_count_total",
 		Help:      "A metrics that counts the number of panics.",
 	})
+
+	PluginEnabled = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: plugin.Namespace,
+		Name:      "plugin_enabled",
+		Help:      "A metric that monitors which plugin(s) is(are) enabled.",
```

Comment:
@jiachengxu , "Indicates whether a plugin is enabled on per server and zone basis".

Reply:
Done